extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -439,6 +445,9 @@ export class AmpA4A extends AMP.BaseElement {
       }
     }
 
+    this.detectedFont_ =
```

Comment:
hmmm... does he mean that we need to wait until after onLayoutMeasure?  Ideally we wouldn't "pollute" A4A for a parameter unique to AdSense impl...